sovrin_client/test/cli/test_tutorial_manual.py

```diff
@@ -274,11 +257,13 @@ def executeGstFlow(name, userCLI, userMap, be, connectedToTest, do, fMap,
                    syncedInviteAcceptedOutWithoutClaims, tMap,
                    transcriptClaimMap)
 
-    # TODO: restart agents to test everything still works fine
-    # faberAgent, acmeAgent, thriftAgent = restartAgents()
+    aliceCLI.looper.runFor(3)
 
-    executeGstFlow("Susan", susanCLI, susanMap, be, connectedToTest, do, fMap,
-                   aMap, jobCertificateClaimMap, newKeyringOut, reqClaimOut,
-                   reqClaimOut1, syncLinkOutWithEndpoint,
-                   syncedInviteAcceptedOutWithoutClaims, tMap,
-                   transcriptClaimMap)
+    # Same flow is executed by different cli
```

Comment:
Yes, that is why I commented it out because it can't work with the changes we made. Susan can't use the same nonce as Alice. So this test is no longer valid.